Lombiq.Tests.UI.Samples/Tests/BasicVisualVerificationTests.cs

```diff
@@ -0,0 +1,50 @@
+using Lombiq.Tests.UI.Attributes;
+using Lombiq.Tests.UI.Constants;
+using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Services;
+using OpenQA.Selenium;
+using System.Drawing;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Lombiq.Tests.UI.Samples.Tests;
+
+// In this basic test we will check visually the rendered content.
+public class BasicVisualVerificationTests : UITestBase
+{
+    public BasicVisualVerificationTests(ITestOutputHelper testOutputHelper)
+        : base(testOutputHelper)
+    {
+    }
+
+    // Checking that everything is OK with the branding of the navbar on the homepage.
+    // For this magic we are using the ImageSharp.Compare package. You can find more info about it here:
+    // https://github.com/Codeuctivity/ImageSharp.Compare/blob/2.0.46/README.md.
+    [Theory, Chrome, VisualVerification]
+    public Task VerifyNavbar(Browser browser) =>
+        ExecuteTestAfterSetupAsync(
+            context =>
+            {
+                // Instead of using context.SetBrowserSize(...), witch is sets the outer size of the browsers window to
+                // the given size, we are using context.SetViewportSize(...) here. This is because the windows borders,
+                // toolbars, tabs and scroll bars usually have different sizes on different platforms/browsers, but we
+                // want the same geometries of rendered content on all platforms.
```

Comment:
Please try out the Grammarly tip I've mentioned, these are all caught by it (or Microsoft Editor):

```suggestion
                // Instead of using context.SetBrowserSize(...), which sets the outer size of the browser's window to
                // the given size, we are using context.SetViewportSize(...) here. This is because the window borders,
                // toolbars, tabs, and scroll bars usually have different sizes on different platforms/browsers, but we
                // want the same geometries of rendered content on all platforms.
```